be/src/vec/exec/volap_scanner.h

```diff
@@ -134,6 +134,7 @@ class VOlapScanner {
     // time costed and row returned statistics
     int64_t _num_rows_read = 0;
```

Comment:
Are `_num_rows_read` and `_raw_rows_read` always equal?

Reply:
I'm not sure about this. But the code for update _num_rows_read and _raw_rows_read seems to be very different.

BTW, if you mean _num_rows_read and _num_rows_return, they are not the same. _num_rows_return is smaller than _num_rows_read, since _num_rows_return is counted after filter_block.